src/applications/disability-benefits/4142/22-4142-schema.json

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
@imeecuison : Why is the JSON schema checked into this directory? Is this to be deleted after you import schema file from vets-json-schema? Additionally, shouldn't the name of file suppose to be 21-4142-schema.json

@Charles-Stone : When do you plan put a PR for schema changes in vets-json-schema repo for the merge?

Reply:
This is generated by the form generator.

Reply:
@imeecuison Do you know if any other forms have been made using this version of the generator?

Reply:
@igreulich : I don't know if other forms have yet been made with the new version of the generator.